src/app/public/plugin-resources/code-examples/timepicker/basic/timepicker-demo.component.html

```diff
@@ -0,0 +1,42 @@
+<form
+  novalidate
+  [formGroup]="formGroup"
+>
+  <sky-timepicker
+    #timepicker
+  >
+    <input
+      formControlName="time"
+      type="text"
+      [skyTimepickerInput]="timepicker"
+    />
+  </sky-timepicker>
+
+  <p *ngIf="timeControl.errors && (timeControl.dirty || timeControl.touched)"
+    class="sky-error-label"
+  >
+    <span [hidden]="timeControl.errors && !timeControl.invalid">
+      Enter a valid time.
+    </span>
+
+    <span *ngIf="timeControl.errors.required">
+      Time is required. Select a value.
+    </span>
+
+    <span *ngIf="timeControl.errors.skyTime">
+      The time is invalid. Enter a valid time or use the picker.
```

Comment:
Is there a reason we have this error message and the previous `Enter a valid time` error message? Seems redundant, so just checking that we are demonstrating 2 distinct errors. Could we just have a single message that said `Select or enter a valid time.'?

Reply:
Good feedback. Done.